desktop/apps/auction2/client.js

```diff
@@ -71,96 +65,39 @@ if (sd.AUCTION && sd.AUCTION.is_live_open == false) {
 }
 
 // Commercial filtering
-const params = new Params(defaultParams, {
-  customDefaults: defaultParams
-})
-const filter = new Filter({ params: params })
+const loggerMiddleware = createLogger()
+
+const store = createStore(
+  auctions,
+  applyMiddleware(
+    thunkMiddleware, // lets us dispatch() functions
+    loggerMiddleware // middleware that logs actions
+  )
+)
 
-// REDUX
-const store = createStore(auctions)
 render(
   <Provider store={store}>
     <HeaderContainer />
   </Provider>,
   document.getElementById('cf-view-filter')
-);
+)
+
+render(
+  <Provider store={store}>
+    <SidebarContainer />
+  </Provider>,
+  document.getElementById('cf-sidebar')
+)
```

Comment:
Oooh, you're right. At this point I could even combine these two into one larger container!